axum-macros/src/from_request.rs

```diff
@@ -23,6 +25,7 @@ impl fmt::Display for Trait {
     }
 }
 
+#[allow(warnings)]
```

Comment:
You probably didn't intend to commit this?

Reply:
Whoops!